src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
+  text = "",
+  italic = false, 
+  bold = false, 
+  underline = false, 
+  size = 'text-base', // uses tailwind utility class to set 1rem
+  color = 'text-black', 
```

Comment:
I would abstract this to a Type class called `TextProps` so people can look up what fields are expected for a Text component

Reply:
I'm going to elaborate on this bc I 100% agree with it. I think as is, this code works but it's a lot to read/looks a little clunky. 

What's being suggested is you essentially do the following: 

type TextProp = {italic: boolean, bold: boolean, ...} 

function Text({italic, bold, ...} : TextProp)  { ...}